app/src/utils/JwtUtil.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Src\utils;
+
+use \Firebase\JWT\JWT;
```

Comment:
@jjsimonAB on top you dont need to specify the \ since you are already on the global scope